docs/documentation/k-Webhooks.md

```diff
@@ -181,3 +181,47 @@ This produces the hash string `feb4b838a272884f6d2c2580b2c7ebb0b2f725b90e8baa6f9
 A replay attack is when an attacker intercepts a valid payload and its signature, then re-transmits them. To mitigate such attacks Smokeball provides a `Timestamp` header which is the ticks representation (.Net format) of when the webhook event was sent. It is also part of the verification signature so an attacker can not change the timestamp without invalidating the signature. 
 
 If the signature is valid but the timestamp is too old, you may choose to reject the message.
+
+## 5. Webhook notification schema
+
+A webhook notification captures changes related to the susbscribed event and includes specific information about the event. Notifications are dependent on the event type.
```

Comment:
Also instead of "Notifications are dependent on the event type." you need to say "Payload is dependant on the event type. See respective API docs for payload examples"